src/roman-numerals/index.js

```diff
@@ -4,6 +4,37 @@
  * @param {string} roman The all-caps Roman numeral between 1 and 3999 (inclusive).
  * @returns {number} The decimal equivalent.
  */
-function romanToDecimal(roman) {}
+const charToNumber = (str) => {
+    switch(str) {
+        case 'I': return 1;
+        case 'V': return 5;
+        case 'X': return 10;
+        case 'L': return 50;
+        case 'C': return 100;
+        case 'D': return 500;
+        case 'M': return 1000;
+        default: return -1;
```

Comment:
Using `-1` makes it seem like it means something. `0` is better - its falsy and wouldn't have any implication on additions.